pages/k8s/troubleshooting.md

```diff
@@ -103,13 +103,15 @@ This will automatically ssh you to the easyrsa unit.
 
 ## Collecting debug information
 
-Sometimes it is useful to collect all the information from a cluster to share with a developer to identify problems. This is best accomplished with
-[CDK Field Agent](https://github.com/charmed-kubernetes/cdk-field-agent).
+To collect comprehensive debug output from your Charmed Kubernetes cluster, install and run
+[juju-crashdump](https://github.com/juju/juju-crashdump) on a computer that has the Juju client installed, with the current controller and model pointing at your Charmed Kubernetes deployment.
 
-Download and execute the collect.py script from
-[CDK Field Agent](https://github.com/charmed-kubernetes/cdk-field-agent) on a box that has a Juju client configured with the current controller and model pointing at the Charmed Kubernetes deployment of interest.
+```bash
+sudo snap install juju-crashdump --channel edge
```

Comment:
Needs `--classic`.  Also, why `edge`?

Reply:
Nice catch on --classic. Edge has something we need, although I can't remember what it is at the moment.